WebDriverAgentLib/FBSpringboardApplication.m

```diff
@@ -99,9 +147,13 @@ - (BOOL)fb_waitUntilApplicationBoardIsVisible:(NSError **)error
 - (BOOL)fb_isApplicationBoardVisible
 {
   [self resolve];
+#if TARGET_OS_TV
+  return self.collectionViews[@"GridCollectionView"].isEnabled;
```

Comment:
is `GridCollectionView` precise?

Reply:
So far, yes since like below...
I need to make sure "HomeBoard" in the application class can use for it, maybe...

![image](https://user-images.githubusercontent.com/5511591/54816994-1d051f80-4cd9-11e9-8589-b38e56512465.png)

Reply:
so far, the `GridCollectionView` looks the most reasonable... at least, it appears on the top...